test/shim_env_command.bats

```diff
@@ -79,5 +79,4 @@ teardown() {
 
   # Should not contain duplicate colon
   run grep '::' <(echo "$path_line")
-  [ "$duplicate_colon" = "" ]
```

Comment:
Why is the assertion removed? explicit assertion is preferred over implicit.

Reply:
The variable `duplicate_colon` isn't set anywhere, and the original [commit](https://github.com/asdf-vm/asdf/commit/549d86e7339110ac4095671c71b9ad3f14b8dd5a) that created this test didn't set it, so I thought it was a mistake

Reply:
Ah okay, well, the result from the `run` command should be checked to ensure that the `::` is not found by grep.

Can you update?

Reply:
Definitely - Gosh I'm kind of embarrassed that I didn't check to replace the test

Reply:
The tests have been missing this for a long, you're just the one discovering it, not your fault!